bundles/org.openhab.binding.goecharger/README.md

```diff
@@ -88,8 +95,9 @@ when
     Item availablePVCurrent received update
 then
     logInfo("Amps available: ", receivedCommand.state)
-    GoEChargerMaxCurrent.sendCommand(receivedCommand.state)
+    MaxCurrTemp.sendCommand(receivedCommand.state)
```

Comment:
This is not the good item name: keep `GoEChargerMaxCurrent` or use `GoEChargerMaxCurrTemp` in case you would like to use the new channel.